src/systems/system.C

```diff
@@ -2096,25 +2107,20 @@ Number System::point_value(unsigned int var, const Point & p, const Elem & e) co
 
   FEType fe_type = dof_map.variable_type(var);
 
-  // Build a FE so we can calculate u(p)
-  UniquePtr<FEBase> fe (FEBase::build(e.dim(), fe_type));
+  // Map the physical co-ordinates to the master co-ordinates using the inverse_map from fe_interface.h.
+  Point coor = FEInterface::inverse_map(e.dim(), fe_type, &e, p);
 
-  // Map the physical co-ordinates to the master co-ordinates using the inverse_map from fe_interface.h
-  // Build a vector of point co-ordinates to send to reinit
-  std::vector<Point> coor(1, FEInterface::inverse_map(e.dim(), fe_type, &e, p));
-
-  // Get the shape function values
-  const std::vector<std::vector<Real>> & phi = fe->get_phi();
+  // create this object to handle the correct
```

Comment:
I think you might be missing a

Reply:
... I think, the important part is removed by markdown?

Reply:
Sorry, I was trying to make a joke.  I say "I think you might be missing a word", but then I omit a word myself, and everyone laughs at the irony?

Anyway, I'm afraid I didn't understand the comment here: I assumed you were typing "to handle the correct shape functions for both FE and InfFE" or some such, but it got cut off somehow.

Reply:
oh, I did not only miss the joke but also the fact that this comment is indeed senseless.